project/Dependencies.scala

```diff
@@ -8,6 +8,8 @@ object Dependencies {
   lazy val awsS3 = "software.amazon.awssdk" % "s3" % awsSdkVersion
   lazy val awsSQS = "software.amazon.awssdk" % "sqs" % awsSdkVersion
   lazy val awsStateMachine = "software.amazon.awssdk" % "sfn" % awsSdkVersion
+  lazy val awsSecretsManager = "software.amazon.awssdk" % "secretsmanager" % awsSdkVersion
+  lazy val playJson = "com.typesafe.play" %% "play-json" % "2.9.4"
```

Comment:
There's already a Json-handling library in the repo, called `upickle`, so this looks to be redundant.